packages/catalog-process/src/services/catalogService.ts

```diff
@@ -1623,16 +1668,4 @@ function assertDailyCallsAreConsistentAndNotDecreased({
   }
 }
 
-function replaceDescriptor(
-  eservice: EService,
-  updatedDescriptor: Descriptor
-): EService {
-  return {
-    ...eservice,
-    descriptors: eservice.descriptors.map((descriptor) =>
-      descriptor.id === updatedDescriptor.id ? updatedDescriptor : descriptor
-    ),
-  };
-}
-
```

Comment:
No longer needed. This was performing exactly the same task as the function that was renamed above (`updateDescriptor -> replaceDescriptor`). We didn't align enough during the merge operations, so both functions have been here for few days.